donut/static/js/groups/positions.js

```diff
@@ -127,3 +128,61 @@ function changePosition(posName) {
     $('#positionSelect').val(posName);
     filterChange();
 }
+
+/*
+ *
+ */
+function groupDelChange() {
+    var groupIndex =$('#groupDel').val();
+    $('#position').find('option').remove();
+    var url = '/1/groups/' + groupIndex + '/positions/';
+    $.ajax({
+        type: 'GET',
+        url: url,
+        success: function(data){
+            for (var i = 0; i < data.length; i++) {
+                var newOption = '<option value=' + data[i].pos_id + '>' + 
+                    data[i].pos_name + '</option>';
+                $('#position').append(newOption);
+            }
+        }
+    })
+}
+
+// Setting up triggers for administration tasks
+$(document).ready(function() {
+    $('#submitPosBtn').click(function(e) {
+        e.preventDefault();
+        $.ajax({
+            type: 'POST',
+            url: '/1/positions/',
+            data: $("#posCreate").serialize(),
+            success: function(data) {
+                p = JSON.parse(data);
```

Comment:
Can set `dataType: 'json'` in AJAX request to get it to automatically interpret it as JSON (also, this would be fixed if you used `jsonify` instead of `json.dumps`)